subprojects/core/src/main/java/org/gradle/internal/cleanup/BuildOutputCleanupRegistryInternal.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.internal.cleanup;
+
+import org.gradle.api.file.FileCollection;
+
+import java.util.Set;
+
+public interface BuildOutputCleanupRegistryInternal extends BuildOutputCleanupRegistry {
```

Comment:
BuildOutputCleanupRegistry is already internal.

Reply:
Right! Fixed